storage/client/file/src/main/java/com/azure/storage/file/models/ShareProperties.java

```diff
@@ -84,7 +86,7 @@ public ShareProperties etag(String etag) {
      *
      * @return the quota value.
      */
-    public int quota() {
+    public long quota() {
```

Comment:
I'm noticing that `ShareGetPropertiesHeaders` had reference types for Integer and Long, whereas this class has primitive types for int and long. Is this intentional? Can we standardise?

Reply:
In the generated code, currently we are generating boxed types if the Swagger defines it as optional - we don't want to throw NullPointerException in our deserialization. The convenience layer can make intelligent defaults but that cannot be automated in the generator.